copyright

```diff
@@ -697,6 +697,9 @@ Files:
  images/land/water8*
  images/land/water12*
  images/land/water13*
+ images/scene/hai?bank*
+ images/scene/hai?shipyard*
+ images/scene/hai?start*
```

Comment:
See #9459. We should have source links to these scenes. Are they actually public domain?

Reply:
they're copied from spaceports and stuff of the game haha

Reply:
What do you mean so they'd load as scenes? A `scene` node in a conversation can reference any image. It doesn't specifically need to be in the scene folder.

Reply:
for the thumbnail of the start iirc if I didnt do this it wouldnt be loaded, and same with the first scenes before you launch but I'm not 100% sure anymore

Reply:
Can you please tell me the names of the images you copied? The copyright for unsplash images is wrong and I am working on fixing it.

Reply:
Thanks, I'll make sure to update that tomorrow

Reply:
NOTE:  If you blindly accept these changes, it will break. The licensing information below this should be moved above these lines.
```suggestion
 images/scene/hai?bank*
Copyright: Tomasz Frankowski
License: public-domain
Comment:
 Taken from https://v.gd/Utf1Ww. This image was uploaded to
 unsplash.com before June 2017, so it is in the public domain.

Files:
 images/scene/hai?shipyard*
Copyright: Dan Asaki
License: Unsplash License
Comment:
 Taken from https://v.gd/UD2WMf. This image was uploaded to
 unsplash.com after June 2017, so it is subject to the
 Unsplash License. It was also uploaded after February 2018,
 so it is subject to an additional restriction limiting the
 sale of unaltered copies.

Files:
 images/scene/hai?start*
Copyright: Willian Justen de Vasconcellos
License: Unsplash License
Comment:
 Taken from https://archive.is/7WLG3. This image was uploaded
 to unsplash.com after June 2017, so it is subject to the
 Unsplash License. It was also uploaded on February 11, 2018,
 so it is not subject to additional restrictions, as this is
 a week before the additional restrictions were added.
```